plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1437,9 +1437,13 @@ public ConnectorMergeTableHandle beginMerge(ConnectorSession session, ConnectorT
     }
 
     @Override
-    public void finishMerge(ConnectorSession session, ConnectorMergeTableHandle tableHandle, Collection<Slice> fragments, Collection<ComputedStatistics> computedStatistics)
+    public void finishMerge(
+            ConnectorSession session,
+            ConnectorMergeTableHandle connectorMergeTableHandle,
+            Collection<Slice> fragments,
+            Collection<ComputedStatistics> computedStatistics)
     {
-        DeltaLakeTableHandle handle = ((DeltaLakeMergeTableHandle) tableHandle).getTableHandle();
+        DeltaLakeTableHandle handle = ((DeltaLakeMergeTableHandle) connectorMergeTableHandle).getTableHandle();
```

Comment:
Rename the one in the ConnectorMetadata interface as well?

Reply:
thanks, will do

Reply:
done